### ISSUE TITLE
Width bigger than normal TextField?

### ISSUE THREAD

**Issue by damianobarbati** [TeamWertarbyte/material-ui-password-field#2]
TextField and PasswordField are defined just the same but password field is wider

```
export default class Auth extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<Flex justify={'space-around'}>
				<Paper>
					<form>
						<Flex flexColumn p={2} align={'center'} justify={'space-around'}>
							<Box>
								<h1>Auth</h1>
							</Box>
							<Box>
								<TextField floatingLabelFixed={true} floatingLabelText="Email" />
							</Box>
							<Box>
								<PasswordField floatingLabelFixed={true} floatingLabelText="Password" />
							</Box>
							<Box>
								<RaisedButton type="submit" primary={true} label="Sign In" />
							</Box>
							<Box>
								<CircularProgress size={36} thickness={3} />
							</Box>
						</Flex>
					</form>
				</Paper>
			</Flex>
		);
	}
}
```

![screen shot 2017-01-27 at 13 26 47](https://cloud.githubusercontent.com/assets/2225869/22370939/6e17d9f8-e494-11e6-8e3e-1cb3032c0d1a.png)

**Comment by leMaik** [comment]
I'll see how I can fix this. Meanwhile, you could try to enable `fullWidth` and see if it fixes the issue.

**Comment by damianobarbati** [comment]
Thanks @leMaik ! I tried to apply fullWidth but I think it is ignored :)

**Comment by leMaik** [comment]
Meh... :cry: Looking into this now.

**Comment by leMaik** [comment]
@damianobarbati Okay, I found the issue. Actually, the `TextField` is smaller than the `PasswordField` and not vice-versa. :smiley: It's limited to 256px (for some reason) if `fullWidth` is not specified, but our `PasswordField` default to no specific width if `fullWidth` is not set to true.

TL;DR: Set `fullWidth` to true for the `TextField` to ensure that it has the same width as the row and thus the same width as the password field.

Please let me know if I could help you.

**Comment by damianobarbati** [comment]
Thank you so much for helping @leMaik.
BTW I'm trying with fullWidth={true} as well but I see different behaviours (same exact containers), no combo results in same width for fields: I'm investigating.

**Comment by leMaik** [comment]
@damianobarbati If you tell me where the `Flex` and `Box` components come from and what they do (flexbox, I guess?), I could support you investigating.

**Comment by damianobarbati** [comment]
@leMaik yes, those are to sketch with flexbox.
Here how I'm using them (I made a stand-alone component to let you test if you want): 

```

import _ from 'lodash';
import React from 'react';
import autobind from 'autobind-decorator';
import { connect } from 'react-redux';

import { Flex, Box } from 'reflexbox';
import Paper from 'material-ui/Paper';
import TextField from 'material-ui/TextField';
import PasswordField from 'material-ui-password-field';
import RaisedButton from 'material-ui/RaisedButton';
import CircularProgress from 'material-ui/CircularProgress';

const email = {
	name: 'email',
	type: 'text',
	required: true,
	pattern: /[^\s@]+@[^\s@]+\.[^\s@]+/,
};

const password = {
	name: 'password',
	type: 'password',
	required: true,
};

export default class Auth extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<Flex sm={12} md={6} justify={'space-around'}>
				<Paper>
					<form onSubmit={this.submit}>
						<Flex flexColumn px={2} align={'center'} justify={'space-around'}>
							<Box>
								<Attribute event={this.event} {...email} />
							</Box>
							<Box>
								<Attribute event={this.event} {...password} />
							</Box>
						</Flex>
					</form>
				</Paper>
			</Flex>
		);
	}
}

class Attribute extends React.Component {
	constructor(props) {
		super(props);
		this.props = props;
		this.state = { error: '' };
  	}

	@autobind
	render() {
		let properties = {
			id: this.props.name, name: this.props.name, type: this.props.type,
			floatingLabelFixed: true, floatingLabelText: _.startCase(this.props.name), hintText: this.props.hint,
			defaultValue: this.props.value,
			errorText: this.state.error, errorStyle: { whiteSpace: 'pre' },
		};

		let input = this.props.type == 'password' ?
			<PasswordField {...properties} /> :
			<TextField {...properties} />;

		return input;
	}
};
```

Thanks for the effort and great job this module! ;)

**Comment by leMaik** [comment]
@damianobarbati Thank you for the standalone example, I hope I'll have time to fix this near the end of the week. 👍

**Comment by leMaik** [comment]
This should be fixed by #10
